### ISSUE TITLE
Request: Option to supress the "This browser does not support Tone.js" message.

### ISSUE THREAD

**Issue by rodw** [Tonejs/Tone.js#439]
This is more of an annoyance than an issue, but I have various unit-tests and other non-browser programs that happen to `require('tone')`in one way or another, which triggers:

```
console.warn("This browser does not support Tone.js");
```

in `Context.js`.  This is a helpful message in general, but I'd like the _option_ to suppress it.

Can you add another variable like `global.TONE_SILENCE_VERSION_LOGGING`, or maybe just piggy-back on that one (since it happens to be used immediately before the warning is printed)?

Thanks!

PS: I could give you a PR for this if you want, but I assume that's more of hindrance than help in this case.

**Comment by tambien** [comment]
We could change that to global.TONE_SILENCE_LOGGING and have it suppress both console.logs. PRs welcome!

**Comment by rodw** [comment]
Cool; thanks @tambien and @stephlow - sorry I wasn't able to get this before anyone else :)

**Comment by dkozar** [comment]
I'm writing some Jest tests. In order to silence down the warning, I used the above approach (`Tone.global.TONE_SILENCE_LOGGING`) in beforeEach . However, it didn't work (seems it's too late at that point in time).

![2019-05-31_20-09-40 copy](https://user-images.githubusercontent.com/1078403/58731157-f0ab0500-83e5-11e9-8208-2d74c9031633.png)

Then I tried to do this in Jest's globalSetup:

```js
module.exports = async () => {
  Tone.global.TONE_SILENCE_LOGGING = true;
};
```

However, I'm getting:

```
ReferenceError: Tone is not defined
```

This means that the Tone library isn't initialised yet, which is good because that's where I want to flip this switch. However, since Tone isn't defined yet, there's no point in doing that. If I mock the Tone.global object, it will be overridden by the library.

I believe that, if the variable would be defined on global object (i.e. window), then it would work:

```js
module.exports = async () => {
  global.TONE_SILENCE_LOGGING = true;
};
```

**Comment by dkozar** [comment]
Oh, I found why the `TONE_SILENCE_LOGGING` doesn't work for me.

The latest released version is [13.4.9.](https://github.com/Tonejs/Tone.js/releases/tag/13.4.9) (see https://github.com/Tonejs/Tone.js/releases). This version still doesn't contain [the fix](https://github.com/Tonejs/Tone.js/pull/453), and this is the version that I picked up from npm by running `yarn add tone`.
